src/pages/start/1.install.mdx

```diff
@@ -19,6 +19,14 @@ The best way to install Nix is to use [Nix installer][nix-installer], a tool fro
 curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
 ```
 
+If you have concerns about the "curl to Bash" approach you can examine the installation script [here][script] and use these steps instead:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix > nix-installer.sh
```

Comment:
I wonder if folks who might prefer this would prefer to download a binary directly? Then can snag the right binary from Github Releases: https://github.com/DeterminateSystems/nix-installer/releases then run it.

Reply:
Good call. I've updated it to include both just in case. Might be a bit overkill but I suspect security-minded folks won't mind.